modules/core/src/passes/layers-pass.js

```diff
@@ -191,6 +193,16 @@ export default class LayersPass extends Pass {
       parent = parent.parent;
     }
 
+    if (layerFilter) {
```

Comment:
- Do we really want to cache layer filtering? Is it that expensive that it is worth the complication? Can't filters be dynamic?
- Can we provide a default `layerFilter: () => true` function so we don't need to have the if statement?